integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/CoupleMBusDevice.feature

```diff
@@ -321,11 +321,11 @@ Feature: SmartMetering Installation - Couple M-Bus Device
       | 8 | unsigned             | 66        |
       | 9 | unsigned             | 3         |
     And the values for classid 72 obiscode "0-2:24.1.0" on device simulator "TEST1024000000001" are
-      | 5 | unsigned             | 3        |
-      | 6 | double-long-unsigned | 12056731 |
-      | 7 | long-unsigned        | 12514    |
-      | 8 | unsigned             | 66       |
-      | 9 | unsigned             | 3        |
+      | 5 | unsigned             | 3         |
+      | 6 | double-long-unsigned | 302343985 |
```

Comment:
I do not get why this is changed to 302343985, 12056731 is the mbus device coupled.....